bin/move_counts_anndata.py

```diff
@@ -44,4 +44,4 @@
     object.uns["X_name"] = "logcounts"
 
     # export object
-    object.write_h5ad(args.anndata_file)
+    object.write_h5ad(args.anndata_file, compression="gzip")
```

Comment:
I think I would make compression optional as an arg to this function, just in case.

I can't suggest the option, but I can implement it here...

```suggestion
    object.write_h5ad(args.anndata_file, compression="gzip" if args.compress else None)
```

I might do this as a "negative" option, so the default is true, which saves you from having to edit anything else:
```python
parser.add_argument(
    "-u",
    "--uncompressed",
    dest="compress",
    action="store_false",
    help="Output an uncompressed HDF5 file",
)
```